src/main/java/FileContent.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Class for reading and writing content from File
```

Comment:
@ArthurRomasyuk Lets skip obvious part, like "class". How about "`@{code Content}`, providing read/write operations to/from `@{code java.io.File}`"?